{"name": "test_data[1]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 1 == 3", "trace": "new_data = 1\n\n    def test_data(new_data):\n        print(\"测试步骤\")\n>       assert new_data == 3\nE       assert 1 == 3\n\nD:\\Python基础\\test\\scripts\\test_fixture_params.py:8: AssertionError"}, "parameters": [{"name": "new_data", "value": "1"}], "start": 1579074481404, "stop": 1579074481407, "uuid": "2195572d-f9ab-42a5-b9ca-89f28a81a6e7", "historyId": "f975f1c8c8ce09339757bdcce8f1d0e0", "testCaseId": "dd465c263251d58de65400669ad5358e", "fullName": "scripts.test_fixture_params#test_data", "labels": [{"name": "parentSuite", "value": "scripts"}, {"name": "suite", "value": "test_fixture_params"}, {"name": "host", "value": "TF-201909261319"}, {"name": "thread", "value": "12284-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "scripts.test_fixture_params"}]}